source/index.html.haml

```diff
@@ -10,11 +10,13 @@
         %li
           %a{href: "/heroku-compatibility"}Heroku compatibility
         %li
-          %a{href: "/internals/procfile"}Procfile - How your app is started
+          %a{href: "/internals/procfile.html"}Procfile - How your app is started
```

Comment:
Actually as strange as it is, the link doesn't work in development, but when deployed it's ok, that's why I've let it like this.